src/whatsnew/3.2.rst

```diff
@@ -113,6 +113,9 @@ Features and Enhancements
   which can include various characters previously not allowed to be included
   in the url info part of endpoint urls.
 
+* :ghissue:`3483`: We added a way of specifying requirements for new user passwords
+  using a list of regexp.
```

Comment:
Let use "list of regular expressions" since some users might not know the shortened version